src/api/api.cpp

```diff
@@ -231,35 +274,66 @@ rgl_mesh_create(rgl_mesh_t *out_mesh, const rgl_vec3f *vertices, int32_t vertex_
 		                         reinterpret_cast<const Vec3i*>(indices),
 		                         index_count).get();
 	});
+	TAPE_HOOK(out_mesh, TAPE_ARRAY(vertices, vertex_count), vertex_count, TAPE_ARRAY(indices, index_count), index_count);
+	return status;
+}
+
+void TapePlay::tape_mesh_create(const YAML::Node& yamlNode)
+{
+	rgl_mesh_t mesh = nullptr;
+	rgl_mesh_create(&mesh,
+		reinterpret_cast<const rgl_vec3f*>(fileMmap + yamlNode["1"].as<size_t>()),
+		yamlNode["2"].as<int>(),
+		reinterpret_cast<const rgl_vec3i*>(fileMmap + yamlNode["3"].as<size_t>()),
+		yamlNode["4"].as<int>());
+	tapeMeshes.insert(std::make_pair(yamlNode["0"].as<size_t>(), mesh));
```

Comment:
Is it possible to address `yamlNode` using integer instead of string? `yamlNode[0]` looks better.

Reply:
Yes, it is possible. Changed